Ginger/Ginger/Actions/ActionEditPage.xaml.cs

```diff
@@ -1786,5 +1796,23 @@ private void OutDSParamType_SelectionChanged(object sender, SelectionChangedEven
             updateDSOutGrid();
             SetDSGridVisibility();
         }
+
+        private void xRawResponseValuesBtn_Click(object sender, RoutedEventArgs e)
+        {
+            if (mAction.RawResponseValues != string.Empty)
+            {
+                string tempFilePath = GingerCoreNET.GeneralLib.General.CreateTempTextFile(mAction.RawResponseValues);
+                if (System.IO.File.Exists(tempFilePath))
+                {
+                    DocumentEditorPage docPage = new DocumentEditorPage(tempFilePath, enableEdit: false, UCTextEditorTitle: string.Empty);
+                    docPage.Width = 800;
+                    docPage.Height = 800;
+                    docPage.ShowAsWindow("Raw Response Output Values");
+                    System.IO.File.Delete(tempFilePath);
+                    return;
+                }
+            }
+            Reporter.ToUser(eUserMsgKey.StaticErrorMessage, "Failed to load raw response view, see log for details.");
```

Comment:
Change error message to "Failed to load raw output values view, see log for details."